spectrocloud/resource_cluster_azure.go

```diff
@@ -268,7 +268,7 @@ func resourceClusterAzure() *schema.Resource {
 							Elem: &schema.Schema{
 								Type: schema.TypeString,
 							},
-							Description: "Availability zones for the machine pool.",
+							Description: "Availability zones for the machine pool. Check if your region provides availability zones on [the Azure documentation](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-service-support#azure-regions-with-availability-zone-support).",
```

Comment:
```suggestion
							Description: "Availability zones for the machine pool. Check if your region provides availability zones on [the Azure documentation](https://learn.microsoft.com/en-us/azure/reliability/availability-zones-service-support#azure-regions-with-availability-zone-support). Default value is `""`",
							Default: "",
```

@nikchern I see this is a string value. I though it was a `list` 🤔

Reply:
Hi @karl-cardenas-coding! Thanks for giving me a hand with this. The type is a Set of String, but it seems like the default value provided has to be of type string. 
```
Type:     schema.TypeSet,
Required: true,
Elem: &schema.Schema{
  Type: schema.TypeString,
 },
```